modules/__tests__/createSpy-test.js

```diff
@@ -73,6 +73,12 @@ describe('A spy', () => {
     expect(spy).toHaveBeenCalledWith(1, 2, 3)
   })
 
+  it('do not treat as equal arguments that contain different instances', () => {
+    // eslint-disable-next-line
+    spy(new String('any'))
+    expect(spy).toNotHaveBeenCalledWith('any')
+  })
+
```

Comment:
This is the key: A `spy` called with `new String('any')` should not `toHaveBeenCalledWith('any')`, for that `'any'` is not the same as `new String('any')` for the purpose of preventing the false positive relevant to this report.